srtcore/api.cpp

```diff
@@ -1123,15 +1132,22 @@ SRTSOCKET CUDTUnited::accept(const SRTSOCKET listen, sockaddr* pw_addr, int* pw_
    {
       // non-blocking receiving, no connection available
       if (!ls->m_pUDT->m_config.bSynRecving)
+      {
+         LOGC(cnlog.Error, log << "srt_accept: no pending connection available at the moment");
```

Comment:
This looks more like a warning. 🤔

Reply:
No. The `srt_accept` function in non-blocking mode should not even be called if the program didn't make sure first that the listener socket has a pending connection in wait.